iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -85,7 +85,8 @@ String processHelmTemplate(String path, String source, HelmInputFileContext inpu
     inputFileContext.setSourceWithComments(sourceWithComments);
     inputFileContext.setAdditionalFiles(helmFilesystem.getRelatedHelmFiles(inputFileContext.inputFile));
     var fileContents = validateAndReadFiles(inputFileContext);
-    return evaluateHelmTemplate(path, inputFileContext, sourceWithComments, fileContents);
+    var path = HelmFileSystem.retrieveHelmProjectFolder(Path.of(inputFileContext.inputFile.uri()), inputFileContext.sensorContext.fileSystem());
+    return evaluateHelmTemplate(String.valueOf(path), inputFileContext, sourceWithComments, fileContents);
```

Comment:
Why do you need to apply `String.valueOf(path)` here?

Reply:
evaluateHelmTemplate expects a String and not a Path as the first argument

Reply:
Please remove this as HelmFilesystem.getFileRelativePath returns a String, not a Path.